includes/templates/template_default/sideboxes/tpl_currencies.php

```diff
@@ -9,8 +9,10 @@
  */
 $content = '';
 $content .= '<div id="' . str_replace('_', '-', $box_id . 'Content') . '" class="sideBoxContent centeredContent">';
+$content .= '<label for="Currency">';
```

Comment:
For the `for=` attribute to be associated with an input/select tag, the attribute's value needs to match the input/select tag's `id=` attribute.  Since the call to `zen_draw_pull_down_menu` doesn't specify an `id=` attribute, that attribute's value will be `select-currency` (noting that case matters).

Note also that there's a Zen Cart function to draw those labels.  Instead of `<label for="Currency">`, use
```php
$content .= zen_draw_label(PLEASE_SELECT, 'select-currency', 'class="sr-only"');
```
_after_ line 13 (so it's in the form) and lose the `</label>` tag on line 17.